schema/manifest_backwards_compatibility_test.go

```diff
@@ -136,14 +136,14 @@ func TestBackwardsCompatibilityManifest(t *testing.T) {
 		//
 		// curl -L -H "Authorization: Bearer ..." -H \
 		// "Accept: application/vnd.docker.distribution.manifest.v2+json" \
-		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc
+		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:d901d5244c94db8151415bf3a1d3a0b10e7468e4997786b01c287cbcbaeca446
```

Comment:
The "Connection established" line is probably [from a proxy][1].  You'll see the 404 if you drop the `head` calls.

[1]: http://stackoverflow.com/questions/16965530/what-to-do-with-extra-http-header-from-proxy